### PR TITLE
Add Quiz feature

### PR DESCRIPTION
**Pull request by ofarrero**
This was created before we agreed to create tests, I will make tests for this at a later date

### CODE REVIEW

File: portfolio/src/main/webapp/quiz.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    answers: {
+      a: 'Computer Science',
+      b: 'Computer Engineering',
+      c: 'Computer Systems',
+    },
+    correctAnswer: 'b'
+  }
+];
+
+function buildQuiz(){
```
Comment (marvel308) [inline portfolio/src/main/webapp/quiz.js:117]: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

File: portfolio/src/main/webapp/quiz.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            </div>`
+        );
+    
+     }
+    );
+    //add question html to quiz html sheet
+    quizContainer.innerHTML = output.join('');
+}
+
+function showResults(){
```
Comment (marvel308) [inline portfolio/src/main/webapp/quiz.js:149]: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

File: portfolio/src/main/webapp/quiz.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+  },
+  {
+    question: 'What am I studying in college?',
+    answers: {
+      a: 'Computer Science',
+      b: 'Computer Engineering',
+      c: 'Computer Systems',
+    },
+    correctAnswer: 'b'
+  }
```
Comment (marvel308) [inline portfolio/src/main/webapp/quiz.js:114]: https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma

File: portfolio/src/main/webapp/quiz.js [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+     myQuestions.forEach( (currentQuestion, questionNumber) => {
+
+        // find selected answer
+        const answerContainer = answerContainers[questionNumber];
+        const selector = `input[name=question${questionNumber}]:checked`; //gets checked answer
+        const userAnswer = (answerContainer.querySelector(selector) || {}).value;
+
+        if(userAnswer === currentQuestion.correctAnswer){
+        // add to correct ans
```
Comment (marvel308) [inline portfolio/src/main/webapp/quiz.js:163]: https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

Comment (marvel308) [reply portfolio/src/main/webapp/quiz.js:163]: Same for else

File: portfolio/src/main/webapp/quiz.js [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+    correctAnswer: 'b'
+  }
+];
+
+function buildQuiz(){
+    const output = [];
+
+    myQuestions.forEach(
+        (currentQuestion, questionNumber)=>{
+        //store all possible answers
```
Comment (marvel308) [inline portfolio/src/main/webapp/quiz.js:122]: https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

Comment (ofarrero) [reply portfolio/src/main/webapp/quiz.js]: Done

File: portfolio/src/main/webapp/quiz.js (hunk 3 above)
Comment (marvel308) [reply portfolio/src/main/webapp/quiz.js:74]: What I meant was you should add a comma after this.

**Pull request merged**